src/template/render.rs

```diff
@@ -452,6 +453,7 @@ impl<'a> ChainTemplateSource<'a> {
         &self,
         context: &TemplateContext,
         command: &[Template],
+        stdin: &Option<Template>,
```

Comment:
Should be `Option<&Template>` instead (you can pass `stdin.as_ref()` to get that). Generally better to accept `Option<&T>` over `&Option<T>` because if a caller has the latter they can make the former, but not vice versa. It doesn't really matter in this case but just a best practice.